### ISSUE TITLE
JS interpolation highlighting

### ISSUE THREAD

**Issue by alexlouden** [AdamCaviness/vs-code-theme-monokai-dark-soda#1]
Firstly, thanks so much for your theme - it's really nice :) 

I've been using it with the following little improvement to highlight JS string interpolation:

```js
// in settings.json
"editor.tokenColorCustomizations": {
  "textMateRules": [
    {
      "scope": "meta.template.expression",
      "settings": {
        "foreground": "#F8F8F2"
      }
    },
    {
     "scope": "punctuation.definition.template-expression.begin, punctuation.definition.template-expression.end",
     "settings": {
       "foreground": "#F92672"
     }
    },
  ]
},
```

Before:
![image](https://user-images.githubusercontent.com/1332954/66626935-97bd6a00-ec2c-11e9-96cf-6bbd76c71181.png)

After:
![image](https://user-images.githubusercontent.com/1332954/66626908-807e7c80-ec2c-11e9-9a49-8de899acc444.png)

Would you be open to accepting a PR adding this change? Thanks!

**Comment by AdamCaviness** [comment]
Absolutely I'd be open to a pull request. Thanks.

**Comment by AdamCaviness** [comment]
Fixed via #2
